tflint/action.yml

```diff
@@ -42,24 +42,28 @@ runs:
       run: tflint --version
       shell: bash
 
+    - name: Set working directory
+      run: |
+        cd ${{inputs.path}}
```

Comment:
the default for "path" is set to "modules/dashboard",
maybe better to set it "/"?